askbot/conf/static_settings.py

```diff
@@ -31,10 +31,9 @@ class AskbotStaticSettings(AppConf):
         }
 
     QUESTION_PAGE_BASE_URL = pgettext('urls', 'question') + '/'
-    print QUESTION_PAGE_BASE_URL
     SERVICE_URL_PREFIX = 's/' # prefix for non-UI urls
     SELF_TEST = True # if true - run startup self-test
-    TRANSLATE_URL = False # set true to localize urls
+    TRANSLATE_URL = True # set true to localize urls
```

Comment:
in case it would be used later; just let it be overwritten by the settings, not by a hard coded value